flask-app/git_push.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
Remove this file?

Reply:
that was a stock generated file . Removed.